modules/op-tkn-pipelines-compatibility-support-matrix.adoc

```diff
@@ -19,18 +23,12 @@ GA:: General Availability
 
 | Operator | Pipelines | Triggers | CLI | Chains | Hub | {pac} | Results | Manual Approval Gate | |
 
+|1.16 | 0.62.x | 0.29.x | 0.38.x | 0.22.x (GA) | 1.18.x (TP) | 0.28.x (GA) | 0.11.x (TP) | 0.3.x (TP) | 4.15, 4.16, 4.17 | GA
```

Comment:
```suggestion
|1.16 | 0.62.x | 0.29.x | 0.38.x | 0.22.x (GA) | 1.18.x (TP) | 0.28.x (GA) | 0.11.x (TP) | 0.3.x (TP) | 4.15, 4.16, 4.17, 4.18 | GA
```
I believe 1.16 we are releasing on 4.18 as well right @ppitonak @vdemeester

Reply:
OpenShift 4.18 is not out yet though. It is planned for the next year, so I don't think we should include it :/

Reply:
Correct, we will support it but it's in early stages in development.